tide/master/resources/js/tide.js

```diff
@@ -825,6 +838,30 @@ function saveSession() {
     });
 }
 
+function setBezel() {
+  $('#wallOutline').css("grid-template-columns", "repeat("+screenCountX +", 1fr)").
+  css("grid-template-rows", "repeat("+screenCountY+", 1fr)").
+  css("grid-column-gap", bezelWidth).css("grid-row-gap", bezelHeight);
+
+  var totalScreens = screenCountX  * screenCountY;
+  var displayPerScreenVertical  = bezelsVertical + 1;
+  var displayPerScreenHorizontal = bezelsHorizontal + 1;
```

Comment:
maybe we could rename all "display" to "monitors" (plural) here to avoid any confusion with the X11 displays?